tests/smoke/support/page/files/allFiles.ts

```diff
@@ -176,7 +202,7 @@ export class AllFilesPage {
     await page.click(`.oc-files-actions-${action}-trigger`)
     await page.click('//ol[@class="oc-breadcrumb-list"]/li/*[1]')
 
-    if (newLocation) {
+    if (newLocation !== 'All files') {
       await cta.files.navigateToFolder({ page: page, path: newLocation })
```

Comment:
can you explain why this is here? Please try to not use translatable string selectors. Now the test only works in english

Reply:
when we want to move/copy to a file in the root directory, than we don't start `cta.files.navigateToFolder`.  "All files" seemed more descriptive of the step
```
And "Brian" copies the following resources
      | resource                | to        |
      | Shares/folder_to_shared | All files |
```
even if you switch the language, the test will work